doc/man/crangler.1

```diff
@@ -15,7 +15,7 @@ The supported transformations are:
 .IP
 Add a contract (pre/post/assigns) to a named C function.
 .IP
-Add a loop invariant to a loop identified by the name of the function its in and
+Add a loop contract to a loop identified by the name of the function its in and
 a loop number.
```

Comment:
While at it: can you also fix the second half of this sentence? I think it should be "name of the function containing the loop and a loop number."